data/dataset_classes/cityscapes_19cl_without_bkgr.txt

```diff
@@ -0,0 +1,19 @@
+road
```

Comment:
please keep initial naming convention (file without background named as <dataset>_<num_classes>cl.txt
and with background - <dataset>_<num_classes>cl_bkgr.txt

Reply:
Done